<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTT Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/monitor.png</normaloff>:/monitor.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuView_V">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionCascade"/>
    <addaction name="actionTile"/>
   </widget>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit_E"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionStart_S"/>
    <addaction name="actionPause_P"/>
    <addaction name="actionStop_E"/>
    <addaction name="separator"/>
    <addaction name="actionOption_O"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
   <addaction name="menuView_V"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Clients</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>11</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Magic</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Show</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Alert</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Color</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Client</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Detect</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Threshold</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last Activity</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Delete</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit_E">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStart_S">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/start.png</normaloff>:/start.png</iconset>
   </property>
   <property name="text">
    <string>Start(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPause_P">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionStop_E">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/export.png</normaloff>:/export.png</iconset>
   </property>
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/about.png</normaloff>:/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/help.png</normaloff>:/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionOption_O">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/option.png</normaloff>:/option.png</iconset>
   </property>
   <property name="text">
    <string>Option(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/cascade.png</normaloff>:/cascade.png</iconset>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/tile.png</normaloff>:/tile.png</iconset>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
